research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -185,107 +344,85 @@ The rules the UA uses to manage these interactions depends on the element types,
-
-
-
-* Applicable to any element.
-* When an element with `delegatesfocus` is focused, the first focusable descendent of that element is focused instead. This is regardless of the element's top-layer status. (This can be added using the [focus delegate spec](https://html.spec.whatwg.org/#focus-delegate).)
-
-The use cases for these attributes need to be better enumerated.
-
-
-## Anchoring (the `anchor` Attribute)
-
-A new attribute, `anchor`, should be added for all elements, whose value is an idref of an “anchoring” element. This anchor relationship is used for two behaviors:
 
 
+## Accessibility / Semantics
 
-1. Establish the provided anchor element as an “ancestor” of this popup, for the purposes of light-dismiss behavior. In other words, when a popup is shown, typically all other popups are closed. An exception would be made for all popups in the ancestor chain formed by the anchor element.
-2. The referenced anchor element could be used by the [Anchor Positioning feature](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/CSSAnchoredPositioning/explainer.md).
+Since the `popup` content attribute can be applied to any element, and this only impacts the element's presentation (top layer vs not top layer), this addition does not have any direct semantic or accessibility impact. The element with the `popup` attribute will keep its existing semantics and AOM representation. For example, `<img popup=popup>` will continue to have `role=img`, but will be able to be displayed on top of other content. Similarly, ARIA can be used to modify accessibility mappings in [the normal way](https://w3c.github.io/html-aria/), for example `<img popup=popup role=presentation>`.
```

Comment:
Swapping around the examples a bit because while I still think they should be valid, using an image as a popup could definitely have its use cases (i'm immediately thinking of "here's where your credit card code is"), but not the common use case I'd imagine people would go for.  

```suggestion
Since the `popup` content attribute can be applied to any element, and this only impacts the element's presentation (top layer vs not top layer), this addition does not have any direct semantic or accessibility impact. The element with the `popup` attribute will keep its existing semantics and AOM representation. For example, `<article popup=popup>...</article>` will continue to be exposed as an implicit `role=article`, but will be able to be displayed on top of other content. Similarly, ARIA can be used to modify accessibility mappings in [the normal way](https://w3c.github.io/html-aria/), for example `<div popup=popup role=note>...</div>`.
```

Reply:
Awesome, thanks. I struggled with a good example here - this is better.